saleor/graphql/product/tests/test_category_filtering.py

```diff
@@ -66,3 +68,388 @@ def test_categories_with_filtering(
     content = get_graphql_content(response)
     categories_nodes = content["data"]["categories"]["edges"]
     assert len(categories_nodes) == categories_count
+
+
+GET_FILTERED_PRODUCTS_CATEGORY_QUERY = """
+    query ($channel: String, $filters: ProductFilterInput){
+      categories(first: 5) {
```

Comment:
Not a big issue but you could just query single category by ID, would make the tests a bit easier. Also to improve readability I would go with given-when-then structure of each test, that's the approach we use in the newly added tests.

Reply:
All test queries have been modified to use single category by ID
All tests were refactored to include given/when/then structure @maarcingebala